src/main/java/github/grace000/erversay/ResponseBuilder.java

```diff
@@ -0,0 +1,39 @@
+package github.grace000.erversay;
+
+import static github.grace000.erversay.Constants.*;
+
+public class ResponseBuilder {
+    private String code = DEFAULT_CODE;
+    private String status = DEFAULT_STATUS;
+    private String body = " ";
+    private int contentLength = 0;
+
+    public ResponseBuilder withCode(String code){
+        this.code = code;
+        return this;
+    }
+
+    public ResponseBuilder withStatus(String status){
+        this.status = status;
+        return this;
+    }
+
+    public ResponseBuilder withBody(String body) {
+        this.body = body;
+        return this;
+    }
+
+    public ResponseBuilder withContentLength(int contentLength, String body) {
+        contentLength = body.getBytes().length;
+        this.contentLength = contentLength;
+        return this;
+    }
+
+    public String getResponse(Response response) {
+        return withCode(response.code).withStatus(response.status).withBody(response.body).withContentLength(response.contentLength, response.body).build();
```

Comment:
Some whitespace here and below on line 37 will make these long lines easier to read.